force-app/main/default/lwc/nksConversationNoteDetails/nksConversationNoteDetails.js

```diff
@@ -51,6 +61,57 @@ export default class NksConversationNoteDetails extends LightningElement {
         }
     }
 
+    @wire(getRecord, {
+        recordId: '$recordId',
+        fields: '$wireFields'
+    })
+    wiredRecordInfo({ error, data }) {
+        if (data) {
+            if (this.objectApiName) {
+                this.getRelatedRecordId('Account.CRM_Person__c', this.objectApiName);
+            }
+        } else if (error) {
+            console.error(error);
+        }
+    }
+
+    @wire(getRecord, {
+        recordId: '$personId',
+        fields: PERSON_ACTORID_FIELD
+    })
+    wiredPersonInfo({ error, data }) {
+        if (data) {
+            const actorId = getFieldValue(data, PERSON_ACTORID_FIELD);
+            this.readAccessToPerson = actorId ? true : false;
+        } else if (error) {
+            console.error(error);
+        }
+    }
+
+    getRelatedRecordId(relationshipField, objectApiName) {
+        getRelatedRecord({
+            parentId: this.recordId,
+            relationshipField: relationshipField,
+            objectApiName: objectApiName
+        })
+            .then((record) => {
+                this.personId = this.resolve(relationshipField, record);
+            })
+            .catch((error) => {
+                console.error(error);
+            });
+    }
```

Comment:
Dette kan være en wire metode i stedet for, der relationshipField er hardkodet til 'Account.CRM_Person__c'